dev/RELEASE_PROCESS.rst

```diff
@@ -18,17 +18,36 @@ This document describes the process for creating new releases.
    by the script above, but they aren't yet.
 
 4. **Increment the Python version:** Create a PR that increments the Python
-   package version. See `this example`_.
+   package version. See `this example`_. You can use the following to get started,
+   assuming you're on a clean master:
+
+   .. code-block:: bash
+
+     NEW_VERSION=0.x.x  # Change this
+     OLD_VERSION=${$(git describe --tags --abbrev=0)#"ray-"}
+     sed -i'.bak' -e "s/$OLD_VERSION/$NEW_VERSION/" python/ray/__init__.py
+     # sed -i "s/$OLD_VERSION/$NEW_VERSION/" python/ray/__init__.py if on Linux
+     git checkout -b release_$NEW_VERSION
+     git add python/ray/__init__.py && git commit -m "Bump version"
+     git push -u [your fork]
+
+     # If you have ``hub`` installed, you can then run:
+     # hub pull-request -m "Bump version from $OLD_VERSION to $NEW_VERSION"
 
 5. **Create a GitHub release:** Create a GitHub release through the `GitHub
    website`_. The release should be created at the commit from the previous
    step. This should include **release notes**. Copy the style and formatting
-   used by previous releases.
+   used by previous releases. Use the following to get started:
+
+   .. code-block:: bash
+
+     git pull origin master --tags
+     git log $(git describe --tags --abbrev=0)..HEAD --pretty=format:"%s" | sort
```

Comment:
What does this do? I've always just made the release through the GitHub UI.

Reply:
This is to generate release docs; I can leave a comment

Reply:
Ok, got it.